lib/msal-node/src/request/ClientCredentialRequest.ts

```diff
@@ -7,7 +7,7 @@ import { CommonClientCredentialRequest } from "@azure/msal-common";
 
 /**
  * CommonClientCredentialRequest
- * - scopes                  - Array of scopes the application is requesting access to.
+ * - scopes                  - An array containing one (string) scope the application is requesting access to.
```

Comment:
```suggestion
 * - scopes                  - Array of scopes the application is requesting access to. Typically contains only the .default scope for a single resource. See: https://learn.microsoft.com/azure/active-directory/develop/scopes-oidc#the-default-scope
```

Reply:
What do you propose we do @derisen ? AAD can only issue 1 token per resource. Maybe we should add an overload that takes only 1 string?

Reply:
This is the case for delegated permissions too though, and we don't call it out specifically -eSTS just ignores the scopes for the 2..n resources and issues a token for the first one only. I feel like like it would be better for STS align the behavior with application permissions with delegated permissions i.e. ignore the other scopes for other resources instead of responding with an error (unless I'm missing something).

What I feel contingent is the single scope/string part, since multiple scopes != multiple resources. It's a single scope because we are using .default with client credentials, but in theory another IdP might allow ["resource1.scope1" and "resource1.scope2"]